### DISCUSSION TITLE
Synchronized simulations

### DISCUSSION THREAD

**Discussion by brunop26** [gama-platform/gama#154] [Q&A]
Hello,
Is there a command that allows to synchronize two simulations launched in parallel and having different settings but the same variables and agents. For example, I run simul1 and simul2, and every 10 cycles, simul2 synchronizes with simul1.
Thanks in advance,

**Comment by lesquoyb** [comment]
Hello Bruno, could you provide us with some concrete example so we get what you are trying to do exactly? In any way, I think @LucasGrjs may have some pointers on simulation synchronization

**Comment by brunop26** [comment]
Hello Baptiste,In the attached example, the two species synchronize every 10 cycles. Can we do the same thing by running two simulations in parallel and synchronizing them every n cycles using a command? Thanks in advance,﻿Bruno PEREZenvoyé : 19 avril 2024 à 06:14de : Baptiste Lesquoy ***@***.***>à : gama-platform/gama ***@***.***>cc : brunop26 ***@***.***>, Author ***@***.***>objet : Re: [gama-platform/gama] Synchronized simulations (Discussion #154) Hello Bruno, could you provide us with some concrete example so we get what you are trying to do exactly? In any way, I think @LucasGrjs may have some pointers on simulation synchronization—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Reply by lesquoyb** [reply]
I think you may have emailed it but this discussion is actually in github's discussion section so I don't see any model attached unfortunately

**Reply by brunop26** [reply]
Hello Baptiste,
The model is attached. 
Thank you.
[Multi-simulation-synchronized.gaml.zip](https://github.com/gama-platform/gama/files/15091591/Multi-simulation-synchronized.gaml.zip)

**Comment by LucasGrjs** [comment]
Hello Bruno, 

I need more clarification on what you want to do when you say  : 
"I run simul1 and simul2, and every 10 cycles, simul2 synchronizes with simul1".

What kind of information or data do you want to exchange between the two simulations?
